packages/fhir-converter/src/lib/handlebars-converter/handlebars-helpers.js

```diff
@@ -209,6 +209,9 @@ var getDateTime = function (dateTimeStringRaw) {
     );
     dateTimeStringRaw = dateTimeStringRaw.toISOString();
   }
+  if (dateTimeStringRaw === null) {
```

Comment:
Could prob move this to the first command inside the function and add ` || dateTimeStringRaw === undefined`)